site/docs/foundations/iconography.mdx

```diff
@@ -3,10 +3,86 @@ title: Iconography
 layout: DetailTechnical
 ---
 
-Icons are visually simplified and recognizable symbols that represent a specific action, object or concept. There are two variant types: Outline and Solid. All icons within the icon set have an outline variant, but not all have a solid variant, for instance, `minimize` (a horizontal dash).
+Icons are simplified visual representations of a specific action, object or concept. Country symbols represent a specific country and are used in contexts where geographical identification is necessary.
+
+The Salt design system has:
+
+- 415 icons
```

Comment:
FYI these numbers will go stale, the next time we add a new icon.
Will we remember to update the numbers and keep this paragraph updated?
If so, great.
Otherwise.. we could say over 500 icons and 200 country symbols or something, or remove, altogether.